SparkPost/2web2ui
# SA-2003: Added fix for filters not properly being unset

### What Changed

- Removing filters properly unsets from query param

### How To Test

- Go to report builder
- Click Add Filters and add at least 1 filter
- Apply
- Verify `query_filters` param exists in URL
- Go to main report builder page
- Remove all filters via tags
- Verify `query_filters` doesn't exist in URL
- Refresh page
- Verify that filters don't repopulate from URL

### To Do
- [ ] Addressed feedback
